zebra-rpc/src/indexer.rs

```diff
@@ -36,3 +41,40 @@ impl BlockHashAndHeight {
             .and_then(|hash| self.height.try_into().ok().map(|height| (hash, height)))
     }
 }
+
+impl BlockAndHash {
+    /// Create a new [`BlockHashAndHeight`] from a [`block::Hash`] and [`block::Height`].
```

Comment:
```suggestion
    /// Creates a new [`BlockAndHash`] from a [`block::Hash`] and [`block::Height`].
```

###### Optional

I tend to follow this RFC for summary sentences: https://rust-lang.github.io/rfcs/1574-more-api-documentation-conventions.html#summary-sentence, so I changed the verb form from imperative to indicative, as suggested in the second paragraph.

Reply:
Applied in https://github.com/ZcashFoundation/zebra/pull/9654/commits/f8c6c5e9523753a9d45199b4d4dc91596bd0c007.

> so I changed the verb form from imperative to indicative, as suggested in the second paragraph

This sounds good to me, I was on the fence, but I didn't know it was suggested in a Rust RFC.

Reply:
I recall reading in some discussions that the imperative form works well for regular comments, contrasting them with doc comments. I tend to follow that.